src/components/Chat/ReplyMessageText.js

```diff
@@ -12,6 +12,7 @@ import {calculateTime} from '../../utils/time';
 import Dot from '../Dot';
 import ActionChat from './ActionChat';
 import ProfileMessage from './ProfileMessage';
+import {COLORS} from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `COLORS` from the theme file is a good change. It replaces hardcoded color values with dynamic ones, improving maintainability and consistency.
```diff
+ import {COLORS} from '../../utils/theme';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->